cves/kernel/CVE-2021-20265.yml

```diff
@@ -413,7 +444,7 @@ lessons:
     applies:
     note:
   secure_by_default:
-    applies:
+    applies: false
```

Comment:
I would argue that this probably applies here. Releasing memory should be a default action. Debateable.